graylog2-server/src/main/java/org/graylog2/periodical/ClusterHealthCheckThread.java

```diff
@@ -35,18 +36,25 @@
     private NotificationService notificationService;
     private final InputRegistry inputRegistry;
     private final NodeId nodeId;
+    private boolean isCloud;
 
     @Inject
     public ClusterHealthCheckThread(NotificationService notificationService,
                                     InputRegistry inputRegistry,
-                                    NodeId nodeId) {
+                                    NodeId nodeId,
+                                    @Named("is_cloud") boolean isCloud) {
         this.notificationService = notificationService;
         this.inputRegistry = inputRegistry;
         this.nodeId = nodeId;
+        this.isCloud = isCloud;
     }
 
     @Override
     public void doRun() {
+        if (isCloud) {
+            LOG.info("Skipping run of ClusterHealthCheckThread, since contained checks are not applicable for Cloud.");
```

Comment:
I think we don't need to log this to info. If we do this in every place we use the flag, we will have a lot of messages at some point. :wink: